tools/individual-packages-gen/gen-phase2.ts

```diff
@@ -0,0 +1,18 @@
+import * as fs from 'fs-extra';
```

Comment:
Would be nice to find better names for these scripts than `gen-phase1` and `gen-phase2`. For example, this one can be called `restore-deps.ts`.

Reply:
Done.